src/main/java/frc/robot/vision/VisionFilters.java

```diff
@@ -0,0 +1,53 @@
+package frc.robot.vision;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import frc.constants.Field;
+import frc.robot.vision.data.AprilTagVisionData;
+import frc.robot.vision.data.VisionData;
+import frc.utils.Filter;
+import frc.utils.ToleranceUtils;
+
+public class VisionFilters {
+
+	public static Filter<VisionData> isPitchAtGoal(Rotation2d wantedPitch, Rotation2d pitchTolerance) {
```

Comment:
```suggestion
	public static Filter<VisionData> isPitchAtAngle(Rotation2d wantedPitch, Rotation2d pitchTolerance) {
```

angle is a thing a motor or something else is trying to get to. This is not a goal. 

same everywhere else.